test-cases/longevity/longevity-lwt-1loader-3h.yaml

```diff
@@ -6,7 +6,6 @@ stress_read_cmd: ["cassandra-stress user profile=/tmp/c-s_lwt_basic.yaml ops'(se
 n_db_nodes: 4
 n_loaders: 1
 n_monitor_nodes: 1
-experimental: true
```

Comment:
Is LWT not marked as experimental anymore?

Reply:
Yes, it's not experimental, as they informed